### ISSUE TITLE
Note in docs about other Pydantic data types

### ISSUE THREAD

**Issue by tiangolo** [tiangolo/fastapi#1597]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * Or, I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Example

N/A. Just docs.

### Description

We should have a link to the Pydantic data types here: https://pydantic-docs.helpmanual.io/usage/types/

From the docs here: https://fastapi.tiangolo.com/tutorial/extra-data-types/

### The solution you would like

A note in that docs page saying something like:

> You can also check all valid the Pydantic data types here: some-link

### Describe alternatives you've considered

N/A.

### Environment

* OS: [e.g. Linux / Windows / macOS]: All (docs)
* FastAPI Version [e.g. 0.3.0]: All (docs)

To know the FastAPI version use:

```bash
python -c "import fastapi; print(fastapi.__version__)"
```

* Python version: All (docs)

To know the Python version use:

```bash
python --version
```

**Comment by tayoogunbiyi** [comment]
I guess I'd make this doc change.

**Comment by tiangolo** [comment]
This was solved by #1612
